tools/testsys/src/run.rs

```diff
@@ -203,10 +226,50 @@ impl Run {
                     .unwrap_or_else(|| "us-west-2".to_string());
                 Box::new(AwsEcsCreator {
                     region,
-                    ami_input: self.ami_input,
+                    ami_input: self.ami_input.context(error::InvalidSnafu {
+                        what: "amis.json is required. You may need to run `cargo make ami`",
+                    })?,
                     migrate_starting_commit: self.migration_starting_commit,
                 })
             }
+            "vmware-k8s" => {
+                debug!("Using family 'vmware-k8s'");
+                let aws_config = infra_config.aws.unwrap_or_default();
+                let region = aws_config
+                    .regions
+                    .front()
+                    .map(String::to_string)
+                    .unwrap_or_else(|| "us-west-2".to_string());
```

Comment:
Probably in the future we could set this as a constant `DEFAULT_REGION`. As a side note, why is `region` needed for `VMware`? Is it to respect the interface defined for testsys?

Reply:
The region is used for SSM calls for VMware migration.

Reply:
This seems like another thing to fix - we should strive to have VMware testing not require any AWS credentials or resources.